core/trino-main/src/main/java/io/trino/execution/QueryManagerConfig.java

```diff
@@ -265,7 +265,8 @@ public int getMaxWriterTasksCount()
         return maxWriterTasksCount;
     }
 
-    @Config("query.max-writer-task-count")
```

Comment:
`task-count` is intentional. https://github.com/trinodb/trino/pull/16238#discussion_r1132334660

Reply:
That doesn't make sense to me.
All the docs, session property, tests and codes are using `tasks-count`.

@sopel39 Could you take a look on this?

Reply:
> It seems the session property wasn't changed when addressing the comment.

Yes, I think addressing of comment was incomplete. @radek-starburst could you update rest of code (docs, session property)